### ISSUE TITLE
Windows 10 and the unability to use a Freedom Scientific braille display through bluetooth

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5276]
**Reported by tyrylu on 2015-08-08 19:46**
Hello,
after the upgrade to Win 10 i'm unable to use the Pac mate 40 blue display with NVDA through bluetooth. The device shows as paired in the device list, but i just get the RuntimeError no device found in the log. Is it causet by Freedom's driver incompatibility or can something be done with that?
Note that an older Focus 40 (not blue) through USB works just fine. Unfortunately, i can't try it with the newer, but it worked in Windows 7 just fine as well.

**Comment by leonardder** [comment]
cc @bramd

**Comment by bramd** [comment]
I was able to use a Focus 40 blue over Bluetooth a few weeks ago in Windows 10, both through the DLL and with my new driver in bramd/nvda/freedomscientific-native branch, which uses the protocol directly.

**Comment by Adriani90** [comment]
@tyrylu are you still having the issue?

**Comment by tyrylu** [comment]
Excluding some october update pairing issues, no, the issue is resolved and likely can be closed.

**Comment by Adriani90** [comment]
Closing as works for me. Thanks for your fast reply.
